contracts/LimitOrderSwap.sol

```diff
@@ -70,6 +70,73 @@ contract LimitOrderSwap is ILimitOrderSwap, Ownable, TokenCollector, EIP712 {
         _fillLimitOrder(order, makerSignature, takerParams, true);
     }
 
+    /// @inheritdoc ILimitOrderSwap
+    function fillLimitOrderGroup(
+        LimitOrder[] calldata orders,
+        bytes[] calldata makerSignatures,
+        uint256[] calldata makerTokenAmounts,
+        address[] calldata profitTokens
+    ) external payable override {
```

Comment:
We won't be validating `msg.value`?

Reply:
Hard to validate cause no info about what amount should be correct.